frontend/src/app.js

```diff
@@ -59,7 +59,7 @@ const App = () => {
   }
 
   useEffect(() => {
-    console.log("Theme changed");
+    // console.log("Theme changed");
```

Comment:
instead of commenting, remove it. Also, make sure it doesn't impact the toast message that we show during the theme change.